test/integration/api/images.bats

```diff
@@ -43,3 +43,16 @@ function teardown() {
 	run docker_swarm images -a
 	[ "${#lines[@]}" -ge 3 ]
 }
+
+@test "docker images -f label" {
+	start_docker_with_busybox 2
+	swarm_manage
+
+	docker_swarm pull dnephin/swarm-integration-test-image-filter-labels:latest
```

Comment:
Sounds good, I wasn't sure which way was better for the suite, or where to put the Dockerfile. I'll make this change now.